modules/dag-l0/src/main/resources/dag-l0.conf

```diff
@@ -12,7 +12,7 @@ snapshot {
     declaration-timeout = 50 seconds
     declaration-range-limit = 3
     lock-duration = 10 seconds
-    peers-declaration-timeout = 10 seconds
+    peers-declaration-timeout = 3 seconds
```

Comment:
This is deliberate; the problem here is there's 3 rounds of this in total; so 10 seconds adds a 30 second delay; since the other 'slow phases' like state transitions are already 5+ seconds, this is a non-trivial amount. This 3 second delay I'm certain of is actually after the unique event change now, as opposed to any resource change.